evm/src/Transceiver/Transceiver.sol

```diff
@@ -72,6 +72,11 @@ abstract contract Transceiver is
         _pause();
     }
 
+    /// @dev unpause the transceiver.
+    function _unpauseTransceiver() internal {
+        _unpause();
+    }
+
```

Comment:
should we just remove the `pause` and `unpause methods on the transceiver?

Reply:
Oh fair yeah it's internal anyway...I was thinking we still want to expose this for integrators, but they can just wrap the underlying `_pause` and `_unpause`

Reply:
yea exactly